ParseTest.Unit/FileStateTests.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using Parse;
+using Parse.Internal;
+using System.Text;
+using System.Collections.Generic;
+using NUnit.Framework;
+
+namespace ParseTest {
+  [TestFixture]
+  public class FileStateTests {

```

Comment:
Can we add a test to `FileTest` to make sure `ParseFile.Url` returns correct `SecureUrl` given 2 kind of test cases:
1. The given `FileState` Uri is secure
2. The given `FileState` Uri is insecure
Since that's what the external developer getting.